package.json

```diff
@@ -23,12 +23,13 @@
     "device": "^0.3.11",
     "express": "^4.16.2",
     "govuk-frontend": "^3.5.0",
-    "hof": "^16.0.0",
+    "hof": "^18.0.0",
     "hof-behaviour-emailer": "^2.2.0",
     "hof-behaviour-summary-page": "^3.1.0",
     "hof-build": "^2.0.0",
     "hof-model": "^3.1.2",
-    "hof-theme-govuk": "^4.1.0",
+    "hof-template-partials": "5.3.5",
+    "hof-theme-govuk": "^5.1.1",
```

Comment:
Not for this PR but I’m sure template partials is really meant to go in another dependency, it might be hof-theme